features/step_definitions/run_steps.rb

```diff
@@ -33,14 +33,15 @@
 Then(/^I get the error$/) do |error_message|
   @error_expected = true
   expect(@last_run_result.error).to be_truthy
-  expect(unformatted_last_run_output.strip).to include(error_message.strip), %(
+  actual = unformatted_last_run_output
+  expect(actual).to include(error_message), %(
```

Comment:
I had to improve the sanitation of output here, because the web view of CircleCI adds a bunch of artifacts into the output that break tests.